### DISCUSSION TITLE
Darkmode as default option in retype.js

### DISCUSSION THREAD

**Discussion by 22h** [retypeapp/retype#218] [Ideas]
hi,
i wanted to ask if it would be possible to set a mode (light/dark) as default in the config. I would like to use retype in a project where everything is dark.

**Comment by geoffreymcgill** [comment]
Hi @22h. Excellent suggestion. At the moment, Retype does not have a config to force dark mode 100% of the time, but it would be simple enough for us to add. 

Better yet, we are expecting to include multiple color palettes as a config in the next release (`v1.12`). One of those options could certainly be a dark palette which maintains the same dark palette in dark mode.

**Comment by qri57i** [comment]
the force of dark mode is implemented?

**Reply by geoffreymcgill** [reply]
> the force of dark mode is implemented?

No, not yet, but we are hoping to include this functionality within the next release.
